modules/homebrew/init.zsh

```diff
@@ -31,3 +31,7 @@ alias caski='brew cask install'
 alias caskl='brew cask list'
 alias casks='brew cask search'
 alias caskx='brew cask uninstall'
+
+if (( $+commands[brew] )) && [[ -d "$(brew --prefix 2>/dev/null)" ]]; then
```

Comment:
Cache the result of `brew --prefix`; don't call it twice. Can you have `brew` without being installed?